source/content/drupal-9.md

```diff
@@ -67,3 +67,16 @@ See the [Drupal 9 Migration Guide](/guides/drupal-9-migration/troubleshoot) for
 ### Where can I report an issue?
 
 [Contact support](/support) to report any issues that you encounter.
+
+### Can I Use Lando or Localdev for Drupal 9
+
+Local development options for Drupal 9 are currently being implemented into [Localdev](/guides/localdev).
+
+Some users have reported success using [Lando](https://docs.lando.dev/basics/) with Drupal 9, but it relies on a workaround and requires extra configuration. Check the status of the [Lando repo's issue](https://github.com/lando/lando/issues/2831#issuecomment-771833900) before you continue.
+
+Manually update the [Lando config file](https://docs.lando.dev/config/global.html#config-yml) to set `drupal9` as the framework:
```

Comment:
Rather than the Lando Config file, we want to direct them to edit the landofile, which is `lando.yml`. The docs for that are at https://docs.lando.dev/config/lando.html#base-file.

Reply:
I'll let the docs experts decide how deep to go, but I wonder if we should suggest creating a project with Lando first. When you create a project from the Pantheon recipe it'll default to `drupal8` for a Drupal 8 or Drupal 9 site. The lando.yml file will include 'framework: drupal8`. After that, a change to `drupal9` and a `lando rebuild` should put them on the right path.

Reply:
thanks, @davidneedham !
I made those changes and tried to incorporate your second comment as an option [L86-88](https://github.com/pantheon-systems/documentation/pull/6474/files#diff-25054decf8627a1f827ee9f686f198e9b17f543d344713f83872a1ba15141b55R86-R88) - telling people what they'll see if they create a new project